Static build error `get_target_property() called with non-existent target "SPIRV-Tools"`

Compiling a static version of glslang with spirv-tools enabled, fails with the following error:
```
CMake Error at C:/projects/glslang/build/tmp/SPIRV-Tools/cmake/SPIRV-ToolsConfig.cmake:3 (get_target_property):
  get_target_property() called with non-existent target "SPIRV-Tools".
Call Stack (most recent call first):
```
which was initial detected at https://github.com/KhronosGroup/glslang/pull/2778#issuecomment-944192494.

The reason for this is that in the static case, the spirv-tools cmake build system creates the targets [`SPIRV-tools-shared`] (https://github.com/KhronosGroup/SPIRV-Tools/blob/06ebc4806b7de62b01b27f2faacd55d639a20180/source/CMakeLists.txt#L374) and [`SPIRV-tools-static`] (https://github.com/KhronosGroup/SPIRV-Tools/blob/06ebc4806b7de62b01b27f2faacd55d639a20180/source/CMakeLists.txt#L383), but refers to the non-existent target `SPIRV-tools` in the generated cmake configuration file (see https://github.com/KhronosGroup/SPIRV-Tools/blob/06ebc4806b7de62b01b27f2faacd55d639a20180/source/CMakeLists.txt#L432)

--- a/source/CMakeLists.txt
+++ b/source/CMakeLists.txt
@@ -428,8 +428,10 @@ if(ENABLE_SPIRV_TOOLS_INSTALL)
   # Special config file for root library compared to other libs.
   file(WRITE ${CMAKE_BINARY_DIR}/${SPIRV_TOOLS}Config.cmake
     "include(\${CMAKE_CURRENT_LIST_DIR}/${SPIRV_TOOLS}Target.cmake)\n"
-    "set(${SPIRV_TOOLS}_LIBRARIES ${SPIRV_TOOLS})\n"
-    "get_target_property(${SPIRV_TOOLS}_INCLUDE_DIRS ${SPIRV_TOOLS} INTERFACE_INCLUDE_DIRECTORIES)\n")
+    "if(TARGET ${SPIRV_TOOLS})\n"
+    "    set(${SPIRV_TOOLS}_LIBRARIES ${SPIRV_TOOLS})\n"
+    "    get_target_property(${SPIRV_TOOLS}_INCLUDE_DIRS ${SPIRV_TOOLS} INTERFACE_INCLUDE_DIRECTORIES)\n"
+    "endif()\n")
   install(FILES ${CMAKE_BINARY_DIR}/${SPIRV_TOOLS}Config.cmake DESTINATION ${PACKAGE_DIR})
 endif(ENABLE_SPIRV_TOOLS_INSTALL)
 

